### DISCUSSION TITLE
هاام جدا

### DISCUSSION THREAD

**Discussion by kreem1986** [community/community#62245] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

ارجو المساعدة فى كيفية اتمام المشروع وانهاء الدورة فاننى توقفت عند هذه المرحلة فهل يمكن المساعدة

**Comment by Imen-Frigui** [comment]
how can i help?

**Reply by kreem1986** [reply]
I finished the exercises as shown in the picture
The certification fee has been paid
But it seemed to me that in order to obtain the certificate, the course must be completed
Indeed, I completed the course, but I stopped building the project and implemented it as required, but I do not know how to finish the project and it appears to me as shown in the pictures
![Screenshot (3)](https://github.com/community/community/assets/140700760/fbf5b9a1-97d4-427d-bf90-2cad126941f2)
![Screenshot (4)](https://github.com/community/community/assets/140700760/bf6ddaf6-64ad-4fdc-95ef-1ea0897153cc)

![Screenshot (5)](https://github.com/community/community/assets/140700760/ba3f9a11-9ec0-4036-b9a4-3f4cd474b20b)

![Screenshot (6)](https://github.com/community/community/assets/140700760/771ef837-ff47-4222-99ee-c8adc6d7489a)

Please help me obtain the certificate of extreme necessity

**Reply by Imen-Frigui** [reply]
you have to submit the URL to the repository not the README
![image](https://github.com/community/community/assets/89009627/0a3692b0-c2f0-421d-b3e9-013e8e8fb5eb)
this is the URL to the repository: https://github.com/kreem1986/SmartHealthAI.git

**Reply by salm255** [reply]
اول مره ادخل هذه الموضوع ولا اعرف عن الكود ولا مساحاته ولا اعرف البرمجه ابدا فارجو المساعده

**Reply by Imen-Frigui** [reply]
المساعدة في ماذا؟

**Comment by kreem1986** [comment]
Thank you for caring
But I entered the address as you mentioned and it appeared to me
![Screenshot (7)](https://github.com/community/community/assets/140700760/5a20f741-89a2-420c-9b3f-ea771ab624cf)

**Reply by Imen-Frigui** [reply]
I see that he is asking for the URL not the HTTPS
https://github.com/kreem1986/SmartHealthAI

**Reply by kreem1986** [reply]
where  URL  please help me

**Reply by Imen-Frigui** [reply, marked as answer]
https://github.com/kreem1986/SmartHealthAI (without the.git)
 have you tried contacting the site about it?

**Reply by kreem1986** [reply]
how contact the site?

**Reply by Imen-Frigui** [reply]
https://community.elementsofai.com/c/building-ai/how-do-i-get-the-credits
https://community.elementsofai.com/c/building-ai/issue-in-downloading-the-certificate

**Comment by kreem1986** [comment]
No one responded to inform me
And the problem still persists

**Reply by Imen-Frigui** [reply]
still the same problem?

**Comment by Ridvan8899** [comment]
يرجى التواصل بشكل عاجل آنا لا اعرف شيء

**Reply by Ridvan8899** [reply]
منذ اسبوع تعرضت لاختراق وخسرت مالي 
والآن ارى هذا البرنامج هنا 
ما هدف هذا التطبيق

**Comment by sondosalbwryny** [comment]
يوجد لدي مشكله في كلمة السر عند دخولي الى برنامج جوليرن كيف رح اقدر افوت عليه

**Comment by Saraserrou** [comment]
Saraserrou

**Comment by Behroz77** [comment]
سلام

در تاریخ پنجشنبه ۲۶ سپتامبر ۲۰۲۴،‏ ۵:۰۲ Saraserrou ***@***.***>
نوشت:

**Comment by salm255** [comment]
الغاء الاشتراك

في الاثنين، ١٦ ديسمبر ٢٠٢٤، ٧:٤١ م Imen Frigui ***@***.***>
كتب:
